flume/neore/rpm/SOURCES/init.d/cygnus

```diff
@@ -25,6 +25,8 @@
 
 PARAM=$1
 
+CYGNUS_INSTANCE=${2}
```

Comment:
Does the script works when `$2` is not provided? Reading issue #332 I undertand it should work that way (if `$2` is not provided them all instances are started/stoped). However, it is not clear in the code... I don't find the point of the code in which you check CYGNUS_INSTANCE is empty or not (maybe it's out there but I'm unable to find :)